lib/Runtime/Types/EdgeJavascriptTypeId.h

```diff
@@ -120,6 +120,7 @@ enum TypeId
     TypeIds_ActivationObject = 73,
     TypeIds_SpreadArgument = 74,
     TypeIds_ModuleNamespace = 75,
```

Comment:
yes, it's a Var can be seen in the code.

---
In reply to: [74170764](https://github.com/Microsoft/ChakraCore/pull/1401#discussion_r74170764) [](ancestors = 74170764)